### PR TITLE
Rename crates to cockpit

### LINKED ISSUE #29: Rename projects to cockpit
- `stackabled` -> `stackable-cockpitd`
- `stackable` -> `stackable-cockpit`
- `stackabled-web` -> `stackable-cockpit-web`

### PR DESCRIPTION
**Pull request by nightkr**
# Description

Fixes #29.

Also organizes the Rust crates into `rust/` (to align with the other Stackable repos), moves libcockpit out of the root folder (which should help avoid silly stuff like including `node_modules/` in its build closure), and moves the Helm wrapper into its own crate.

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Review by Techassi — CHANGES_REQUESTED** [review]
Mostly minor stuff, otherwise lgtm. Nice job!

File: default.nix [hunk 1]
```diff
@@ -29,23 +29,23 @@ rec {
         LIBCLANG_PATH = "${pkgs.libclang.lib}/lib";
         BINDGEN_EXTRA_CLANG_ARGS = "-I${pkgs.glibc.dev}/include -I${pkgs.clang.cc.lib}/lib/clang/${pkgs.lib.getVersion pkgs.clang.cc}/include";
       };
-      stackabled-web = attrs: {
+      stackable-cockpit-web = attrs: {
```
[the remark this answers is not in the document]
Comment (nightkr) [reply default.nix:32]: Depends on whether you think of it as "the web component of cockpitd" or "the web frontend for cockpit", I suppose...

File: rust/helm-sys/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+name = "helm-sys"
+version = "0.1.0"
+edition = "2021"
+publish = false
+links = "helm"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
```
[the remark this answers is not in the document]
Comment (nightkr) [reply rust/helm-sys/Cargo.toml]: 0612cb98424064398c3935944f3db8abd76c28e0

File: README.md [hunk 3]
```diff
@@ -63,8 +64,8 @@ hooks are:
 - `cargo xtask gen-comp`: Runs shell completions generation for `stackablectl`
 - `cargo xtask gen-mam`: Runs man page generation for `stackablectl`
 
-[server-readme]: ./bins/stackabled/README.md
-[ctl-readme]: ./bins/stackablectl/README.md
+[server-readme]: ./rust/stackable-cockpitd/README.md
+[ctl-readme]: ./rust/stackablectl/README.md
 [pre-commit]: https://pre-commit.com/
 [web-readme]: ./web/README.md
 [lib-readme]: ./src/README.md
```
[the remark this answers is not in the document]
Comment (nightkr) [reply README.md]: 2dd73fc302d138b59be1b70539c14989a38bbc70

File: README.md [hunk 4]
```diff
@@ -30,18 +31,18 @@ cd stackable-cockpit
...
 Each component can be build separately like this:
 
 ```shell
-cargo build --release -p stackablectl             # Builds stackablectl
-cargo build --release -p stackabled               # Builds the Stackable API server
-cargo build --release -p stackabled --features ui # Builds the Stackable API server bundled with the admin UI
-cd web && pnpm build && cd -                      # Builds the admin UI
+cargo build --release -p stackablectl                    # Builds stackablectl
+cargo build --release -p stackable-cockpit               # Builds the Stackable Cockpit API server
+cargo build --release -p stackable-cockpit --features ui # Builds the Stackable Cockpit API server bundled with the admin UI
```
[the remark this answers is not in the document]
Comment (nightkr) [reply README.md]: 2dd73fc302d138b59be1b70539c14989a38bbc70

File: README.md [hunk 5]
```diff
@@ -30,18 +31,18 @@ cd stackable-cockpit
 ```
 
 The admin UI is registered as a crate and is part of the build process, as the HTML/CSS/JS bundle is included in the
-final `stackabled` binary. To get the build process running, first execute `pnpm i` to install all required NodeJS
+final `stackable-cockpitd` binary. To get the build process running, first execute `pnpm i` to install all required NodeJS
```
[the remark this answers is not in the document]
Comment (nightkr) [reply README.md]: 2dd73fc302d138b59be1b70539c14989a38bbc70

File: default.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (Techassi) [reply default.nix:32]: Yeah... I don't have a super strong opinion on this. Let's just leave it as is.

**Review by Techassi — APPROVED** [review]
Approved.

A little off-topic: We might want to add `yarn install` to the web build script as it otherwise fails to run various actions, like rustdoc or clippy. We can also add the conditional to only run `yarn install` if node_modules is not present.

**Pull request merged**